index.html

```diff
@@ -30,16 +45,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box one">Box 1</div>
+                <div class="box two">Box 2</div>
+                <div class="box three">Box 3</div>
+                <div class="box four">Box 4</div>
+                <div class="box five">Box 5</div>
+                <div class="box six">Box 6</div>
+                <div class="box seven">Box 7</div>
+                <div class="box eight">Box 8</div>
+                <div class="box nine">Box 9</div>
+                <div class="box ten">Box 10</div>
```

Comment:
Nice job with this. I'm glad you used another class. Most people used an id which works too, but your way is more of the convention.

The only thing I would nitpick about is the naming of the other class. For something small like this its okay. But inside of just naming it one, two, three, four, and so on. I would've liked to see something like `box1`, `box2`, `box3`. It's just a bit more descriptive. Then you can just style it like

```
.box1 {
<--code here !-->
}
``` 
Instead of having to fight with specificity like
```
.boxes .box.one {
    <--code here !-->
}
```